config/msb/conf.d/camera.yaml

```diff
@@ -1,10 +1,17 @@
-msb-camera:
-  verbose : false
-  print_stdout : false
-  fps : 10               # max 30
-  width : 1920           # 1080p
-  height : 1080          # 1080p
-  rollover_interval : 60 # every 60 seconds a new video file will be opened
-  video_dir : /home/pi/msb_data/cam
+verbose : false
+serial_number: "msb-devel"
+print_stdout : false
+fps : 10                # max 30
+width : 1920            # 1080p
+height : 1080           # 1080p
+rollover_period : 600 # every 60 seconds a new video file will be opened
+video_dir : "/home/msb/msb_data/cam"
```

Comment:
Maybe name the dir the same as the package, i.e. .../camera

Reply:
done